aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -477,6 +504,9 @@ REGISTER_DISPATCH(clamp_min_stub, &clamp_min_kernel);
 IMPLEMENT_COMPLEX_KERNEL(FLOATING, acos)
 IMPLEMENT_COMPLEX_KERNEL(FLOATING, asin)
 IMPLEMENT_COMPLEX_KERNEL(FLOATING, atan)
+// IMPLEMENT_COMPLEX_KERNEL(FLOATING, acosh)
```

Comment:
Best to just remove these lines.

Reply:
Done, I removed for others as well (sinh, abs, etc.). I hope that's fine.